### PR TITLE
Add Datera CloudStack Storage Plugin 4.10.0.0

### PR DESCRIPTION
**Pull request by jcshen007**
1.Create/Delete Dynamic Primary Storage
2.VM create with root disk on Datera
3.Create Datera native snapshots
4.Create template from snapshots
5.Create volume from Snapshots
6.Storage volume QoS.
7.Volume resizing with QoS change.

### CODE REVIEW

**Comment by jcshen007** [comment]
Submit a new PR after rebased to the latest upstream/master

**Comment by rhtyd** [comment]
@jcshen007 can you rebase against latest master and fix the version to 4.11.0.0-SNAPSHOT etc?

**Comment by rhtyd** [comment]
Is this still WIP or experimental/PoC or you would want some feedback on this @jcshen007

**Comment by rafaelweingartner** [comment]
Ping @jcshen007

File: plugins/storage/volume/datera/test/org/apache/cloudstack/storage/datastore/driver/DateraPrimaryDataStoreDriverTest.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    @Before
+    public void setUp() throws Exception {
+
+        //setup Mock objects
+        when(mockHost.getStorageUrl()).thenReturn(STORAGE_URL);
+
+    }
+
+    @Test
+    public void testGrantAccess() throws Exception {
```
Comment (rafaelweingartner) [inline plugins/storage/volume/datera/test/org/apache/cloudstack/storage/datastore/driver/DateraPrimaryDataStoreDriverTest.java]: Are you planning to really implement the test cases?!?
If not, please remote these classes.

**Comment by rhtyd** [comment]
Ping @jcshen007 any update, where the integration stands? Can you rebase to latest master?

**Comment by jcshen007** [comment]
Closing the outdated PR.

**Pull request closed without merging**